iso15118/shared/exi_codec.py

```diff
@@ -378,11 +380,18 @@ def from_exi(
                 return msg_class.parse_obj(msg_dict)
 
             raise EXIDecodingError("Can't identify protocol to use for decoding")
-        except ValidationError as exc:
-            raise EXIDecodingError(
-                f"Error parsing the decoded EXI into a Pydantic class: {exc}. "
-                f"\n\nDecoded dict: {decoded_dict}"
-            ) from exc
+        except V2GMessageValidationError as exc:
```

Comment:
I dont think you can just substitute the Pydantic error `ValidationError` for the `V2GMessageValidationError` as there are errors that Pydantic will throw as `ValidationError` because we tried to parse an integer into a field that was supposed to be a str for example...

Reply:
Done